Himanshu_Kumar/project1-(calculator)/assets/css/cal_css.css

```diff
@@ -0,0 +1,46 @@
+.cal-div {
+  background-color: rgba(128, 128, 128, 0.329);
+  box-shadow: 10px 10px 20px rgba(128, 128, 128, 0.781);
+  border-radius: 25px 25px 5px 5px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 500px;
+  
+}
+.input-container{
+  width: 100%;
+  height: 120px;
+  border: none;
+  border-radius: 25px 25px 0px 0px;
+  background-color: rgba(0, 0, 0, 0.815);
+  padding: 30px 20px 10px;
+
+}
+
+.cal-input {
+  width: 100%;
+  height: 70px;
```

Comment:
I am seeing a lot of use of px unit in CSS, this is not what I recommended to use and also explained why not to use it

@Hhumour This is giving me an expression all this code this mostly copy-pasted

Reply:
With continuous trial and error, i might have forgot to mention change it back to %. But my target was to set it in % at the end.

Reply:
always check each line of your code before opening a PR, this will help you to stay away from `red`comments